CREDITS

```diff
@@ -0,0 +1,2 @@
+ownCloud Android (GNU GPL v2.0 https://github.com/owncloud/android/blob/master/LICENSE.txt)
+    Part of Sharing-related code in ShareActivity
```

Comment:
I really didn't find any guides on how to make credits, so taken general approach. Will move to activity code, but when we will have >1 credits - better to move them to separate file, like this one.